tests/test_watchmaker.py

```diff
@@ -185,45 +129,21 @@ def test_extra_arguments_list(provider_mock):
 
 
 @patch("watchmaker.status.Status.initialize", return_value=None)
-def test_extra_arguments_map(provider_mock):
+def test_extra_arguments_map(_provider_mock):
     """Test map in extra_arguments loads correctly."""
     # setup
     raw_arguments = {
         "extra_arguments": [
-            '--user-formulas',
-            '{"foo-formula": "https://url"}'
+            "--user-formulas",
+            '{"foo-formula": "https://url"}',
         ]
     }
     check_val = {"user_formulas": {"foo-formula": "https://url"}}
 
-    watchmaker_arguments = watchmaker.Arguments(**dict(**raw_arguments))
+    watchmaker_arguments = watchmaker.Arguments(**raw_arguments)
 
     # test
     watchmaker_client = watchmaker.Client(watchmaker_arguments)
 
     # assertions
     assert watchmaker_client.worker_args == check_val
-
-
-@patch("watchmaker.status.Status.initialize", return_value=None)
-def test_none_arguments_aws_provider(provider_mock):
```

Comment:
This test actually is doing something... It is validating that the `Arguments()` invocation is converting string values of `"None"` to the pythonic none value `None`...

Reply:
But it is no different then test_none_arguments test, it does the exact same thing. which is why I removed it.

Reply:
Ahh, I see